src/background.js

```diff
@@ -41,23 +41,33 @@ const tearDown = (tabId = null) => {
   });
 };
 
+// Store per-tab setup listeners to keep kinase active on refresh
+const updatedListeners = {};
```

Comment:
Don't think so, since this is the background page it will stick around until quitting Chrome and it's really not needed elsewhere!